src/utils.jl

```diff
@@ -301,6 +302,45 @@ end
 sparse_init(dims...; kwargs...) = sparse_init(Random.GLOBAL_RNG, dims...; kwargs...)
 sparse_init(rng::AbstractRNG; init_kwargs...) = (dims...; kwargs...) -> sparse_init(rng, dims...; init_kwargs..., kwargs...)
 
+"""
+    truncated_normal([rng=GLOBAL_RNG], dims...; μ = 0, σ = 1, a = -2., b = 2.)
```

Comment:
Do we want `μ, σ, a, b` as keywords? Elsewhere I think the goal is never to require anyone to type greek letters. And I think `a,b` are much too obscure, `left, right` or `lo, hi` seem more obvious.

```suggestion
    truncated_normal([rng=GLOBAL_RNG], dims...; mean = 0, std = 1, lo = -2, hi = 2)
    truncated_normal([rng]; kw...) = (dims...) -> truncated_normal(rng, dims...; kw...)
```
I think this should mention the curried form somehow. But perhaps writing it out like this is too verbose?

Reply:
I'll fix the Greek letters 👍🏽 The curried form hasn't been mentioned for the others, so I just followed precedent - I can change that if it seems more appropriate